components/NavigationContainer/index.js

```diff
@@ -0,0 +1,20 @@
+import { StyledNavigationContainer } from "./NavigationContainer.style";
+
+export default function NavigationContainer({
+  top,
+  left,
+  right,
+  bottom,
+  children,
+}) {
+  return (
+    <StyledNavigationContainer
+      top={top}
+      left={left}
+      right={right}
+      bottom={bottom}
+    >
+      {children}
+    </StyledNavigationContainer>
+  );
+}
```

Comment:
This higher order function just serves the purpose of renaming your component, you can directly use StyledNavigationContainer where you currently use NavigationContainer

Reply:
I don't understand it. where should I use the styledNavigationContainer directly? 
I need it as a component for navigating through my app (like on ./pages/myanalyses or ./pages/index.js)

Reply:
Take a look at this simple example of an unnecessary higher order function:

https://codesandbox.io/s/gracious-dust-4spdks?file=/src/App.js

Reply:
thank you! i tried to adapt it and to write the styling directly where i use the navigation container etc. 🙏🏻